apps/judicial-system/web/src/components/InfoCard/useInfoCardItems.tsx

```
@@ -338,19 +338,23 @@ const useInfoCardItems = () => {
-          : strings.reviewTagAppealed,
-        {
-          isFine:
-            workingCase.indictmentRulingDecision ===
-            CaseIndictmentRulingDecision.FINE,
-        },
-      ),
-    ],
+    values: workingCase.defendants
+      ? workingCase.defendants.map((defendant) => (
+          <Text>
+            {`${defendant.name} - ${formatMessage(
```

Biome (2.3.13)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)